ChiTech/ChiPhysics/PhysicsMaterial/transportxsections/prop_transportxs_06_export.cc

```diff
@@ -51,21 +51,22 @@ void chi_physics::TransportCrossSections::
   if (num_precursors>0)
     ofile << "NUM_PRECURSORS " << num_precursors << "\n";
 
-  Print1DXS(ofile,"SIGMA_T"   ,sigma_tg);
-  Print1DXS(ofile,"SIGMA_F"   ,sigma_fg  ,1.0e-20);
+  Print1DXS(ofile, "SIGMA_T"   , sigma_t);
+  Print1DXS(ofile, "SIGMA_F"   , sigma_f  , 1.0e-20);
+  Print1DXS(ofile, "SIGMA_A"   , sigma_a  , 1.0e-20);
   Print1DXS(ofile,"NU"        ,nu        ,1.0e-20);
   Print1DXS(ofile,"NU_PROMPT" ,nu_prompt ,1.0e-20);
   Print1DXS(ofile,"NU_DELAYED",nu_delayed,1.0e-20);
-  Print1DXS(ofile,"CHI_PROMPT",chi_g     ,1.0e-20);
-  Print1DXS(ofile,"DDT_COEFF" ,ddt_coeff ,1.0e-20);
+  Print1DXS(ofile, "CHI_PROMPT", chi     , 1.0e-20);
```

Comment:
this bugs me. see previous comments. there should be chi, chi-prompt and chi-delayed.

Reply:
This will be addressed via issue #146